### PR TITLE
Report Manager Enhancement and fix bugs

### PR DESCRIPTION
**Pull request by guyyoo**

### CODE REVIEW

File: AppBuilder/platform/views/viewComponent/ABViewReportsManagerComponent.js [hunk 1]
```diff
@@ -393,7 +393,6 @@ module.exports = class ABViewReportsManagerComponent extends ABViewComponent {
 
                   switch (r.type) {
                      case "date":
-                     case "datetime":
```
Comment (nh758) [inline AppBuilder/platform/views/viewComponent/ABViewReportsManagerComponent.js:396]: Why did you remove `datetime`?

**Review by nh758 — APPROVED** [review]

File: AppBuilder/platform/views/viewComponent/ABViewReportsManagerComponent.js (hunk 1 above)
Comment (guyyoo) [reply AppBuilder/platform/views/viewComponent/ABViewReportsManagerComponent.js:396]: on `view: "query"`, the `datetime` type is `date`. Previously, the`datetime` type of selection criteria did not change to `date` types.
https://docs.webix.com/api__ui.query_fields_config.html
